TrashMob/client-app/src/components/Partners/Partnerships.tsx

```diff
@@ -128,8 +128,8 @@ export const Partnerships: FC<any> = () => {
                             Supplied services from a given partner may vary by location/branch.{' '}
                         </p>
                     </Col>
-                    <Col sm={5}>
-                        <Image src={Garbage} alt='garbage bags being picked up' className='mh-100 mt-0' />
+                    <Col sm={5} className='d-flex align-items-middle'>
```

Comment:
align-items-middle class doesn't exists, maybe you mean "items-center" ?

Reply:
My mistake!